feat: create 2021-11-09-hacktoberfest-risultati (#1101)
italia/developers.italia.it: _posts/it/2021-11-09-hacktoberfest-risultati.md

---
layout: post
title: "Hacktoberfest 2021: i risultati"
subtitle: Developers Italia ha partecipato all’ottava edizione dell’evento internazionale sull’open source
date: 2021-11-09 17:00:00 +0200

> tags: developers api opensource

Reviewer: ```suggestion
tags: hacktoberfest developers api opensource
```
There's also the `hacktoberfest` tag https://developers.italia.it/it/news/hacktoberfest

categories: news
author: Francesco Pognante
image: /assets/icons/logo-it.png
locale: it
---
Anche quest’anno abbiamo *hackerato* ottobre! Developers Italia ha partecipato all’ottava edizione dell’[Hacktoberfest](https://hacktoberfest.digitalocean.com/){:target="_blank"}, iniziativa internazionale volta a promuovere la comunità e lo sviluppo di software open source tramite una serie di sfide. Tra castagne, zucche di Halloween e foliage, gli sviluppatori di tutto il globo hanno affrontato più di cento “problemi” (issue) aperti su vari repository contenuti all’interno dell’organizzazione [github.com/italia](https://github.com/italia){:target="_blank"}.

> Com’è andata la festa? Sono state  **[27 le issue risolte](https://github.com/search?q=label%3Ahacktoberfest+org%3Aitalia+updated%3A%3E2021-08-01+state%3Aclosed&type=Issues&ref=advsearch&l=&l=)**{:target="_blank"}: 10 di queste hanno migliorato il sito di [Developers Italia](https://github.com/italia/developers.italia.it){:target="_blank"} con correzioni grafiche e ottimizzazioni al motore di ricerca del [Catalogo del software](https://developers.italia.it/it/search){:target="_blank"}. Ben 17 hanno contribuito all’evoluzione del [design-react-kit](https://github.com/italia/design-react-kit/){:target="_blank"}, con risoluzione di bug, ampliamento della documentazione e perfino aggiunta di nuovi componenti di UI. Un grande thank you a tutti quelli che, da tutto il mondo, hanno partecipato, con un messaggio in una issue, un suggerimento o una Pull Request: ognuno ha contribuito alla causa a suo modo.  

Reviewer: ```suggestion
Com’è andata la festa? Sono state  **[27 le issue risolte](https://github.com/search?q=label%3Ahacktoberfest+org%3Aitalia+updated%3A%3E2021-08-01+state%3Aclosed&type=Issues&ref=advsearch&l=&l=)**{:target="_blank"}: 10 di queste hanno migliorato il sito di [Developers Italia](https://github.com/italia/developers.italia.it){:target="_blank"} con correzioni grafiche e ottimizzazioni al motore di ricerca del [Catalogo del software](https://developers.italia.it/it/search){:target="_blank"}. Ben 17 hanno contribuito all’evoluzione del [design-react-kit](https://github.com/italia/design-react-kit/){:target="_blank"}, con risoluzione di bug, ampliamento della documentazione e perfino aggiunta di nuovi componenti di UI. Un grande *thank you* a tutti quelli che, da tutto il mondo, hanno partecipato, con un messaggio in una issue, un suggerimento o una Pull Request: ognuno ha contribuito alla causa a suo modo.  
```

> **Vuoi continuare a divertirti e imparare a sviluppare in modo aperto con i progetti di Developers Italia?** Nessun problema, con noi la festa dura tutto l’anno: tutti i software, le librerie e i kit di sviluppo che permettono di interagire con le Piattaforme Abilitanti del Paese sono al 100% open source ed è possibile migliorare insieme questi software con un approccio aperto e collaborativo. Prova [https://come-partecipo.italia.it/](https://come-partecipo.italia.it/){:target="_blank"}.

Reviewer: ```suggestion
**Vuoi continuare a divertirti e imparare a sviluppare in modo aperto con i progetti di Developers Italia?** Nessun problema, con noi la festa dura tutto l’anno: tutti i software, le librerie e i kit di sviluppo che permettono di interagire con le Piattaforme Abilitanti del Paese sono al 100% open source ed è possibile migliorare insieme questi software con un approccio aperto e collaborativo. Scopri come partecipare sul nostro sito [https://come-partecipo.italia.it/](https://come-partecipo.italia.it/){:target="_blank"}.
```

Author: "Nostro sito" non rischia di confondersi con Developers?

Reviewer 2: @Francescopog :+1:

Metterei Scopri come partecipare su come.partecipo.italia.it

Author: ok mi fai la suggestion tu o modifico il file originario dopo?

Reviewer: Si scusa era solo un suggerimento, vedi tu la forma corretta

> **Vuoi continuare a divertirti e imparare a sviluppare in modo aperto con i progetti di Developers Italia?** Nessun problema, con noi la festa dura tutto l’anno: tutti i software, le librerie e i kit di sviluppo che permettono di interagire con le Piattaforme Abilitanti del Paese sono al 100% open source ed è possibile migliorare insieme questi software con un approccio aperto e collaborativo. Prova [https://come-partecipo.italia.it/](https://come-partecipo.italia.it/){:target="_blank"}.

Reviewer 2: ```suggestion
**Vuoi continuare a divertirti e imparare a sviluppare in modo aperto con i progetti di Developers Italia?** Nessun problema, con noi la festa dura tutto l’anno: tutti i software, le librerie e i kit di sviluppo che permettono di interagire con le Piattaforme Abilitanti del Paese sono al 100% open source ed è possibile migliorare insieme questi software con un approccio aperto e collaborativo. Scopri come partecipare su [https://come-partecipo.italia.it/](https://come-partecipo.italia.it/){:target="_blank"}.
```

Nel nostro catalogo, inoltre, si possono trovare i [software a riuso](https://developers.italia.it/it/search?type=software_reuse&sort_by=relevance&page=0){:target="_blank"} e i [software open source](https://come-partecipo.italia.it/){:target="_blank"} di terze parti inseriti a catalogo dalle Pubbliche Amministrazioni italiane o da singoli sviluppatori per essere riutilizzati da altri Enti pubblici. Anche in questo caso è possibile dare una mano agli sviluppatori per migliorare i servizi pubblici digitali.
 

> Ti aspettiamo, per maggiori approfondimenti sul nostro [Slack](http://developersitalia.slack.com){:target="_blank"} e su [Forum Italia](https://forum.italia.it/){:target="_blank"}.

Reviewer: ```suggestion
Ti aspettiamo, per maggiori approfondimenti sul nostro [Slack](https://developersitalia.slack.com){:target="_blank"} e su [Forum Italia](https://forum.italia.it/){:target="_blank"}.
```
